### ISSUE TITLE
Deprecation warnings with numpy 1.20

### ISSUE THREAD

**Issue by jacklovell** [cherab/core#281]
Numpy 1.20 has deprecated `np.bool`, `np.int` etc: https://numpy.org/devdocs/release/1.20.0-notes.html#deprecations

This means many Cherab scripts now display the following warning:
```
DeprecationWarning: `np.bool` is a deprecated alias for the builtin `bool`. To silence this warning, use `bool` by itself. Doing this will not modify any behavior and is safe. If you specifically wanted the numpy scalar type, use `np.bool_` here.
```
These deprecated aliases are used in the following places in Cherab core:
```
(cherab) jlovell@freia028 Cherab> grep -E -r "np\.int[^0-9]" --include="*.py" --include="*.pyx" --include="*.pxd" cherab_core
cherab_core/cherab/tools/tests/test_raytransfer.py:        voxel_map_ref = -1 * np.ones((4, 10), dtype=np.int)
cherab_core/cherab/tools/tests/test_raytransfer.py:        voxel_map = -1 * np.ones((4, 10), dtype=np.int)
cherab_core/cherab/tools/tests/test_raytransfer.py:        voxel_map_ref = -1 * np.ones((4, 10, 10), dtype=np.int)
cherab_core/cherab/tools/tests/test_raytransfer.py:        voxel_map = -1 * np.ones((4, 10, 10), dtype=np.int)
cherab_core/cherab/tools/tests/test_raytransfer.py:        voxel_map_ref = -1 * np.ones((10, 10, 10), dtype=np.int)
cherab_core/cherab/tools/tests/test_raytransfer.py:        voxel_map = -1 * np.ones((10, 10, 10), dtype=np.int)
cherab_core/demos/ray_transfer/4_ray_transfer_map.py:voxel_map = np.zeros((100, 100), dtype=np.int)
```
```
(cherab) jlovell@freia028 Cherab> grep -E -r "np\.bool[^0-9]" --include="*.py" --include="*.pyx" --include="*.pxd" cherab_core                                                                                    
cherab_core/cherab/tools/tests/test_raytransfer.py:        mask = np.zeros((4, 10), dtype=np.bool)
cherab_core/cherab/tools/tests/test_raytransfer.py:        mask_ref = np.zeros((4, 10), dtype=np.bool)
cherab_core/cherab/tools/tests/test_raytransfer.py:        mask = np.zeros((4, 10, 10), dtype=np.bool)
cherab_core/cherab/tools/tests/test_raytransfer.py:        mask_ref = np.zeros((4, 10, 10), dtype=np.bool)
cherab_core/cherab/tools/tests/test_raytransfer.py:        mask = np.zeros((10, 10, 10), dtype=np.bool)
cherab_core/cherab/tools/tests/test_raytransfer.py:        mask_ref = np.zeros((10, 10, 10), dtype=np.bool)
cherab_core/cherab/tools/raytransfer/emitters.pyx:            mask = mask.astype(np.bool)
cherab_core/cherab/tools/raytransfer/emitters.pyx:            mask = np.ones(self.grid_shape, dtype=np.bool)
```

These deprecated aliases should be replaced with the built-in Python types, `int` and `bool` respectively, as per the deprecation guidelines.

**Comment by vsnever** [comment]
Thanks, I added this code, so I'll fix it.

**Comment by vsnever** [comment]
Fixed in #282.
